ctapipe/image/cleaning.py

```diff
@@ -532,6 +602,43 @@ def __call__(
         )
 
 
+class TimeCleaner(ImageCleaner):
+    """
+    Clean images using the time clustering cleaning method
+    """
+
+    space_scale_m = FloatTelescopeParameter(
+        default_value=0.25, help="Pixel space scaling parameter in m"
```

Comment:
Can we determine the scale parameters from the informatino in `CameraDescription`?

Reply:
I am not sure. That parameter needs optimization, it could 2x or 3x the pixel spacing of the camera and teh optimized number depends on the telescope